### PR TITLE
Add nepali-date-converter extension

### PR DESCRIPTION
**Pull request by Beedhan**
## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
What about making it a `no-view` command, which updates the subtitle of the command instead? That would look like this :)

image

I used `updateCommandMetadata({ subtitle: bsDate });` in the `no-view` command

**Comment by Beedhan** [comment]
I've made the requested changes. Thank you for the suggestion.

File: extensions/nepali-date-converter/package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "icon": "extension_icon.png",
+  "author": "beedhan",
+  "categories": [
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Convert to BS",
```
Comment (pernielsentikaer) [inline extensions/nepali-date-converter/package.json:15]: Is there a name that suits better?

Comment (Beedhan) [reply extensions/nepali-date-converter/package.json:15]: It should have been B.S. which is an abbreviation of Bikram Samwat. I think "Convert to Nepali Date" or "Convert to Nepali Calendar" would be better.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**